create.sh

```diff
@@ -116,10 +121,11 @@ sudo cp base/restart.sh ~atsign/atsign/etc/renewal-hooks/deploy
     tput setaf 2
 # It would be nice to use the @sign for the name but
 # Docker insists on a name that is DNS compliant and so emojis and @ signs are out hence the $SERVICE tag
-# we use a neat trick using docker-compose to create the compose file for us.
+# we use a neat trick usign docker-compose to create the compose file for us.
     echo Starting secondary for $ATSIGN at $FQDN on port $PORT as $DNAME on Docker
-sudo -u atsign docker-compose --env-file ~atsign/dess/$ATSIGN/.env -f ~atsign/dess/$ATSIGN/docker-swarm.yaml config | sudo -u atsign tee ~atsign/dess/$ATSIGN/docker-compose.yaml > /dev/null
-sudo -u atsign docker stack deploy -c ~atsign/dess/$ATSIGN/docker-compose.yaml $SERVICE
+
+sudo docker-compose --env-file ~atsign/dess/$ATSIGN/.env -f ~atsign/dess/$ATSIGN/docker-swarm.yaml config | sudo -u atsign tee ~atsign/dess/$ATSIGN/docker-compose.yaml > /dev/null
```

Comment:
This creates an incorrect docker-compose.yaml because the volumes get defined as /root/atsign rather than /home/atsign

The root cause of this issue lies in the use of ~/atsign/... in dess/base/docker-swarm.yaml where it might be preferable to use ~atsign/atsign/... (and in general @cconstab it might be worth cleaning up, or at least explaining why files get put into ~atsign/atsign ~atsign/base and ~atsign/dess)

Reply:
So the issue would be fixed with introduction of / at the ~/atsign?

Reply:
Either `/home/atsign/atsign` or `~atsign/atsign`